### DISCUSSION TITLE
Mirth Connect 4.5.2 with Entra ID Authentication to SQL

### DISCUSSION THREAD

**Discussion by sgurumurthy2** [nextgenhealthcare/connect#6381] [Q&A]
Hello, 

I am unable to start the mirth connect service with SQL MI MFA authentication. Any input here will be helpful. 

Mirth Version : 4.5.2
SQL Managed Instance - Azure
Working: Below connection string works with SQL authentication account. 

jdbc:jtds:sqlserver://sqlmi-aadpillart1-dev-centralus-repairstack.49cad5d99aa7.database.windows.net/mirthmisqldb-aadpillart1-Dev-centralus-transition;ssl=request

Not working: 
jdbc:jtds:sqlserver://sqlmi-aadpillart1-dev-centralus-repairstack.49cad5d99aa7.database.windows.net/mirthmisqldb-aadpillart1-Dev-centralus-transition;domain=MMM;useNTLMv2=true;ssl=request

Error: 

ERROR 2024-12-19 20:12:40.828 [Main Server Thread] com.mirth.connect.server.Mirth: Error establishing connection to database, aborting startup. Login failed. The login is from an untrusted domain and cannot be used with Integrated authentication.
INFO  2024-12-19 20:12:40.828 [Shutdown Hook Thread] com.mirth.connect.server.Mirth: shutting down mirth due to normal request

Direct access from SSMS works as expected.

Thanks

**Comment by pacmano1** [comment]
jtds is deprecated.

**Reply by sgurumurthy2** [reply]
Thank you.  I am new to Mirth Connect and this is my first time setting it up. I am referring to an old install. What are my alternatives for connecting to Azure SQL with Entra ID. Any recommendation is appreciated.

**Comment by kirbykn2** [comment]
Use the MS SQL driver.  Maybe something like:
jdbc:sqlserver://<server-name>.database.windows.net:1433
;database=<database-name>;authentication=ActiveDirectoryIntegrated
Look at the difference between connecting to the db from SSMS and
connecting from Mirth.


On Thu, Dec 19, 2024 at 4:29 PM pacmano1 ***@***.***> wrote:

> jtds is deprecated.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nextgenhealthcare/connect/discussions/6381#discussioncomment-11622219>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APRXWD7JIMSRVZDD4KUEK5D2GM3CFAVCNFSM6AAAAABT53COGGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNRSGIZDCOI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <nextgenhealthcare/connect/repo-discussions/6381/comments/11622219@
> github.com>
>


-- 
Best,

Kirby Knight
| 231.735.4650 | ***@***.***

**Reply by pacmano1** [reply]
Honestly with MFA, not sure how that would work with a java "service".

**Comment by AcuteParanoia** [comment]
@sgurumurthy2 Did you ever get any resolution on this?  It appears I'm going to run into this same exact situation with Azure and Managed Identities.
